Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "crypto-primes"
-version = "0.7.0-pre.1"
+version = "0.7.0-dev"
```

Comment:
Curious: why would you change the version back to `-dev` after each release?

That blocks anyone from pulling unreleased changes from git and try them out.

Reply:
Sure, but that's kind of the point: test out the impact of your api change before release.